glsl-piglit/fetch-piglit.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+git clone --depth=1 https://gitlab.freedesktop.org/mesa/piglit.git
```

Comment:
Maybe this could be a git submodule instead?

Reply:
I feel like turning this into a submodule would make it look like more of a "required" dependency, and considering this is a massive repository I like having it be optional.

Maybe if the piglit test suite proves useful for testing this crate it'll make sense to turn it into a submodule and make it part of the default suite of tests. I'm curious to see how other parser libraries deal with this kind of situation tbh.

Reply:
Agreed.